Graph-iOS-Swift-Snippets/ApplicationConstants.swift

```diff
@@ -6,28 +6,31 @@
 
 import Foundation
 
-
-struct ApplicationConstants {
+struct ApplicationConstants
+{
     // You will set your application's clientId
-    static let clientId = "619a3897-5a78-4a84-b5a8-4bcd22100f29"
+    static let clientId = "YOUR CLIENT ID"
```

Comment:
Should we add some client ID here so sample works?

Reply:
In general, no. We try to be consistent across Graph samples - the instructions explain how to obtain a client id.